main.go

```diff
@@ -285,42 +290,51 @@ func main() {
 
 	decoder := admission.NewDecoder(mgr.GetScheme())
 
+	// TODO handle case where auto monitor is enabled but auto annotation config is not specified
+	var autoAnnotationConfig auto.AnnotationConfig
+	var autoAnnotationMutators *auto.AnnotationMutators
+	supportedLanguages := instrumentation.SupportedTypes()
+
 	if os.Getenv("DISABLE_AUTO_ANNOTATION") == "true" || autoAnnotationConfigStr == "" {
```

Comment:
Can we introduce a similar env var to disable AutoMonitor on the Operator directly?